bokeh/palettes.py

```diff
@@ -1224,13 +1473,31 @@
     "Cividis" : Cividis,
 }
 
-colorblind = {
-    "Colorblind" : Colorblind
+tol = {
+    "Bright": Bright,
+    "HighContrast": cast(Dict[int, Any], HighContrast),
```

Comment:
I'm not sure why this cast should be necessary, but I think it might be possible to fix this by explicitly specifying a type where HighContrast is defined, rather than a cast here (and I'd regard that as marginally better)

Reply:
Without the cast, mypy raises this error when I add HighContrast to dictionaries like `accessible` or `all_palettes`: 

> incompatible type "Dict[str, object]"; expected "Mapping[str, Dict[int, Tuple[str, ...]]]"

Mypy infers the type based on the first assignment it reads. We defined `Bright = { 3: Bright3, 4: Bright4, 5: Bright5, 6: Bright6, 7: Bright7 }`, which has a type of `Dict[int, Tuple[str, ...]]`. However, for `HighContrast = { 3: HighContrast3 }`, it has a type of `Dict[int, Tuple[str, str, str]]`, which mypy thinks they are different. So when we add HighContrast to `accessible` and `all_palettes`, it became `Dict[str, object]` instead of `Mapping[str, Dict[int, Tuple[str, ...]]]`.

The only way I can think of solving this is by casting... I can cast the original HighContrast with `Dict[int, Tuple[str, ...]]` instead of `Any` if that's preferred.? I'm a bit new to python too so it will be greatly appreciated if you could guide me on how to exactly solve this 🙏

Reply:
I was suggesting something like this (but, I have not actually tried it)
```
HighContrast: Dict[int, Tuple[str, ...]] = { 3: HighContrast3 }
```

Reply:
Thanks! I think it's working :)